### DISCUSSION TITLE
我不太理解计算器的返回值

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#126] [Q&A] [answered]
如题，我没有太明白ppt中计算器返回的是什么，如果返回的是字符串，那么字符串没有=这一操作符。我是否需要新定义一个class并重载赋值来作为计算器的返回值类型
![image](https://user-images.githubusercontent.com/100075314/158000648-d5e0a63e-2c82-4e7f-b76f-6628c78429c5.png)

**Comment by binbin00** [comment, marked as answer]
string
